tests/migrations/test_commands.py

```diff
@@ -616,6 +617,20 @@ def test_sqlmigrate_for_non_atomic_migration(self):
             self.assertNotIn(connection.ops.start_transaction_sql().lower(), queries)
         self.assertNotIn(connection.ops.end_transaction_sql().lower(), queries)
 
+    @override_settings(MIGRATION_MODULES={"migrations": "migrations.test_migrations_simple_atomic_migration"})
+    def test_sqlmigrate_for_non_transcational_databases(self):
+        """
+        Transaction wrappers aren't shown for databases not supporting transactional DDL.
+        """
+        out = io.StringIO()
+        connection.can_rollback_ddl = False
+        call_command("sqlmigrate", "migrations", "0001", stdout=out)
+        output = out.getvalue().lower()
+        queries = [q.strip() for q in output.splitlines()]
+        if connection.ops.start_transaction_sql():
```

Comment:
You could assign `start_transaction_sql` to a variable to avoid calling it twice.